packages/amplify-migration-tests/src/__tests__/update_tests/storage_migration_update.test.ts

```diff
@@ -89,28 +91,23 @@ describe('amplify add/update storage(S3)', () => {
   });
 
   afterEach(async () => {
-    // await deleteProject(projRoot);
-    // deleteProjectDir(projRoot);
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot);
   });
 
   it('init a project and add s3 bucket & update with new trigger', async () => {
     // init, add storage and push with local cli
     await initJSProjectWithProfile(projRoot, {});
-    await addS3Storage(projRoot);
+    await addAuthWithDefault(projRoot, {});
+    await addS3StorageWithAuthOnly(projRoot);
```

Comment:
```suggestion
    await addS3StorageWithAuthOnly(projRoot, {});
```

will you not get an undefined if you don't send in a setting?

Reply:
Nvm was looking at the wrong function